src/rest/StreamEndpoints.js

```diff
@@ -60,52 +45,70 @@ export async function createStream(props, apiKey = this.options.apiKey) {
         apiKey,
         {
             method: 'POST',
-            body: JSON.stringify(props),
+            data: props,
         },
     )
     return json ? new Stream(this, json) : undefined
 }
 
 export async function getOrCreateStream(props, apiKey = this.options.apiKey) {
-    let json
-
-    // Try looking up the stream by id or name, whichever is defined
-    if (props.id) {
-        json = await this.getStream(props.id, apiKey)
-    } else if (props.name) {
-        json = await this.getStreamByName(props.name, apiKey)
-    }
+    let stream = await (props.id ? this.getStream(props.id, apiKey) : this.getStreamByName(props.name, apiKey))
```

Comment:
Is this code really more readable?

Also it is not equivalent to the old code (new code queries `getStreamByName` even if `props.name` is not defined, which doesn't make sense).